tests/functional/a/arguments_differ.txt

```diff
@@ -4,3 +4,4 @@ arguments-differ:41:4:ClassmethodChild.func:Parameters differ from overridden 'f
 arguments-differ:68:4:VarargsChild.has_kwargs:Parameters differ from overridden 'has_kwargs' method
 arguments-differ:71:4:VarargsChild.no_kwargs:Parameters differ from overridden 'no_kwargs' method
 arguments-differ:172:4:SecondChangesArgs.test:Parameters differ from overridden 'test' method
+arguments-differ:239:4:ChildClass2.meth:Parameters differ from overridden 'meth' method
```

Comment:
Probably not related to the current MR but that would be nice to have:
```suggestion
arguments-differ:239:4:ChildClass2.meth:Parameter 'arg' was of type 'int' in 'ParentClass' and is now a 'bool' in overridden 'ChildClass2.meth' method 
```
Could we do something similar in argument-renamed ?

Reply:
Nice idea, but at that case I think we would have multiple cases for *args, **kwargs, argument removals, additions etc. Maybe a bit complicated? For the arguments-renamed it's more simple, maybe we could say for example :
`arguments-renamed:200:4:ChildClass2.meth:Parameter 'arg'  in 'ParentClass' has been renamed to 'arg1' in overridden 'ChildClass2.meth' method`